scripts/test/Muon/muon_period_info_widget_test.py

```diff
@@ -0,0 +1,61 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import unittest
+
+from mantidqt.utils.qt.testing import start_qapplication
+from Muon.GUI.Common.muon_period_info_widget import MuonPeriodInfoWidget, CYCLES_NOT_FOUND, DAQ, DWELL, NOT_DAQ_STRING
+
+
+@start_qapplication
+class MuonPeriodInfoWidgetTest(unittest.TestCase):
+    def setUp(self):
+        self.widget = MuonPeriodInfoWidget()
+
+    def assert_row_values(self, row_num, period_number, name, type, daq_number, frames, total_frames, counts, tag):
+        self.assertEqual(period_number, self.widget._table.item(row_num, 0).text())
+        self.assertEqual(name, self.widget._table.item(row_num, 1).text())
+        self.assertEqual(type, self.widget._table.item(row_num, 2).text())
+        self.assertEqual(daq_number, self.widget._table.item(row_num, 3).text())
+        self.assertEqual(frames, self.widget._table.item(row_num, 4).text())
+        self.assertEqual(total_frames, self.widget._table.item(row_num, 5).text())
+        self.assertEqual(counts, self.widget._table.item(row_num, 6).text())
+        self.assertEqual(tag, self.widget._table.item(row_num, 7).text())
+
+    def test_empty_on_initialize(self):
+        self.assertEqual(True, self.widget.is_empty())
+
+    def test_clear(self):
+        self.assertEqual(True, self.widget.is_empty())
```

Comment:
This test is the same as test_empty_on_initialize , did you want to add rows then call clear method then check if empty ?